### PR TITLE
Mention of -DSTACK_TRACE_ON_USEREXCEPTION when printing UserException.

### PR DESCRIPTION
**Pull request by jean-philippe-martin**

### CODE REVIEW

**Comment by lbergelson** [comment]
closing this since it's a duplicate of #2443

**Comment by jean-philippe-martin** [comment]
@lbergerson, you closed my pull request because it's a duplicate of your issue report? I think it's #2445 you meant to close. I closed it for you.

**Comment by droazen** [comment]
Re-opening

**Comment by droazen** [comment]
@lbergelson Will this property work when running GATK via `gatk-launch`?

File: src/main/java/org/broadinstitute/hellbender/Main.java [hunk 1]
```diff
@@ -184,6 +184,8 @@ protected void handleUserException(Exception e) {
 
         if(printStackTraceOnUserExceptions()) {
             e.printStackTrace();
+        } else {
+            System.out.println("Use -DSTACK_TRACE_ON_USEREXCEPTION to print the stack trace.");
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/Main.java]: Should this go to stderr?

**Comment by lbergelson** [comment]
@jean-philippe-martin I was holding off on this because because we're still talking about adding this as a special argument and removing the property.

**Comment by jean-philippe-martin** [comment]
Sure; this isn't particularly urgent so take your time.

File: src/main/java/org/broadinstitute/hellbender/Main.java (hunk 1 above)
Comment (jean-philippe-martin) [inline src/main/java/org/broadinstitute/hellbender/Main.java]: I don't have a strong opinion either way, I can change the code if we have consensus we prefer stderr.

**Comment by droazen** [comment]
@jean-philippe-martin My thoughts on this are that since this property exists now, we might as well mention to the user that it exists by merging this PR, until such time as the replacement for this functionality is implemented. Could you make the stderr change, and then merge this? Thanks!

**Comment by jean-philippe-martin** [comment]
Changed to stderr. Waiting for the tests as per good policy.

**Pull request merged**